draft-hodges-webauthn-registries.xml

```diff
@@ -274,7 +288,7 @@ https://github.com/w3c/webauthn/pulls?q=is%3Apr+label%3Aspec%3Awebauthn-registri
         </t>
         <t>
           Registered extension identifiers MUST be a maximum of 32 octets in length and MUST
-          consist only of printable USASCII characters, excluding backslash and doublequote,
+          consist only of printable ASCII characters, excluding backslash and doublequote,
```

Comment:
not using USASCII seems kinda silly given this quote from RFC 20:

> The standard code may be identified by the use of the notation ASCII
or USASCII.

..but whatever...